schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/2-0-0

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a data load succeeding",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitoring.batch",
+    "name": "load_succeeded",
+    "format": "jsonschema",
+    "version": "2-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "shredding": {
```

Comment:
@chuwy I guess you use the word `shredding`to match `shredding_complete`.  When is the right time to change these schemas to `transforming`?

Reply:
That's a good question. I _think_ it's when we start using "tarnsformer" out in the wild - right now it's just our plans and the change could be more confusing than helping. Perhaps it would be better to do in one big bang release: remove all references to "shredder" and use new schemas.

One good counter-point would be that if we change it now - data will remain in the same table for longer time, but to me it doesn't seem strong enough.

Anyways, if you think now is the time - I can change that.

Reply:
No strong opinion on this - was just giving you the opportunity to consider it.  I'll resolve this conversation.